app/reducers/devices.js

```diff
@@ -1,12 +1,14 @@
 import mm723Image from '../../images/MM723_CNL_combo@2x.jpg';
+import mm600Image from '../../images/MM600_CNL_combo@2x.jpg';
+
 const devices = {
   carelink: {
     instructions: ['Import from CareLink', '(We will not store your credentials)'],
     isFetching: false,
     key: 'carelink',
     name: 'Medtronic',
     // for the device selection list
-    selectName: 'Medtronic (from CareLink)',
+    selectName: 'Medtronic (CareLink import)',
```

Comment:
@gniezen FYI, https://trello.com/c/BF1URAvH covers the copy changes in this file that are not related to 600-series.

Reply:
As CareLink is hidden now, this specific change doesn't matter. Removing the `-` for 5/7-series LGTM.